config.go

```diff
@@ -114,6 +114,9 @@ type Config struct {
 	LookupdPollInterval time.Duration `opt:"lookupd_poll_interval" min:"5s" max:"5m" default:"60s"`
 	LookupdPollJitter   float64       `opt:"lookupd_poll_jitter" min:"0" max:"1" default:"0.3"`
 
+	// Duration between reconnect attempts when connecting directly to nsqd (i.e not using nsqlookupd)
+	ReconnectInterval time.Duration `opt:"reconnect_interval" min:"0" max:"1m" default:"15s"`
```

Comment:
I dislike having separate variables for the reconnect interval using nsqd directly vs using lookupd. It's a shame the var for lookupd is named as such because this is probably a better name.

do you feel comfortable just re-using `LookupdPollInterval` ?